src/jit/importer.cpp

```diff
@@ -11226,6 +11226,23 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 #endif // _TARGET_64BIT_
                 }
 
+#ifdef FEATURE_CORECLR
+                if ((oper == GT_ADD) || (oper == GT_SUB) || (oper == GT_MUL) || (oper == GT_DIV) || (oper == GT_MOD))
```

Comment:
I thought about that but since only those operation mentioned in the table, I am checking only for them.

How to determine that operation is unsigned?